onboarding/mentors.md

```diff
@@ -21,33 +21,38 @@ It's up to the mentoring pair to find what works best for them and to ensure the
 ## How to mentor
 *Moving from the abstract to the concrete:*
 
+### Before your first day
+- Become intimately familiar with the contents of the [onboarding checklist](https://github.com/artsy/potential/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md).
+- Identify issues in Sentry that are appropriate for a first task that can be pushed to production during their first week.
```

Comment:
I would like to expand on this in either this doc or in a separate doc that we link to from here. @joeyAghion would you be able to help me understand how to identify these types of issues?